integrationtests/wildfly-modules/src/test/java/org/infinispan/test/integration/as/client/BaseHotRodQueryIT.java

```diff
@@ -68,4 +68,31 @@ public void testRemoteQuery() throws Exception {
 
       rcm.stop();
    }
+
+   @Test
+   public void testUninverting() throws Exception {
```

Comment:
It's maybe not trivial to figure out what this test is doing, maybe put a comment like "Using a sort on a field that does not contain DocValues so Lucene is forced to uninvert it" to the posterity?